### ISSUE TITLE
Cannot build React app using nano-memoize

### ISSUE THREAD

**Issue by tokland** [anywhichway/nano-memoize#3]
Cannot build react app using nano-memoize (v1.0.0):

```
$ yarn build
yarn run v1.7.0
$ node scripts/build.js
Creating an optimized production build...
Failed to compile.

Failed to minify the code from this file: 

 	../my-project/node_modules/nano-memoize/index.js:3 

Read more here: http://bit.ly/2tRViJ9
```

The reference says:

> Some third-party packages don't compile their code to ES5 before publishing to npm. This often causes problems in the ecosystem because neither browsers (except for most modern versions) nor some tools currently support all ES6 features. We recommend to publish code on npm as ES5 at least for a few more years.

**Comment by anywhichway** [comment]
If we had a larger dev team, we would perhaps take on managing a transpiled code base. Since we don't and most developers are running a build pipeline, we have chosen not to publish transpiled code at this point. In the worst case we have found that transpiled code does not actually work (async generators are sometimes subject to this), creates hard to debug bloated code and expands the scope of performance testing needs. We recommend you do your development and initial testing using a modern browser and Node v9 or greater; then, include babel in your build pipeline and transpile your entire code base.

This being said, if you would like to collaborate on this project, add a build process to it that includes transpilation and copies files appropriately to something like nano-memoize.es5.js; then, we are certainly open to it!

**Comment by silverwind** [comment]
This was fixed in `react-scripts@2.0.0`, see https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify.

**Comment by anywhichway** [comment]
@silverwind Thanks for taking the time to let us know. Closing this issue.
